masonry_winit/src/event_loop_runner.rs

```diff
@@ -45,46 +45,34 @@ impl From<accesskit_winit::Event> for MasonryUserEvent {
     }
 }
 
+/// The state of the current app.
 #[expect(unnameable_types, reason = "TODO")]
-#[allow(
-    clippy::large_enum_variant,
-    reason = "we don't have that many instances of it"
-)]
-pub enum WindowState {
-    Uninitialized(WindowAttributes),
-    Rendering {
-        handle: Arc<WindowHandle>,
-        accesskit_adapter: Adapter,
-    },
-    Suspended {
-        handle: Arc<WindowHandle>,
-        accesskit_adapter: Adapter,
-    },
-}
-
-impl Debug for WindowState {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        match self {
-            Self::Uninitialized(attrs) => f.debug_tuple("Uninitialized").field(attrs).finish(),
-            Self::Rendering { .. } => f.debug_struct("Rendering").finish_non_exhaustive(),
-            Self::Suspended { .. } => f.debug_struct("Suspended").finish_non_exhaustive(),
-        }
-    }
+#[derive(Default, Debug)]
+pub enum AppState {
```

Comment:
Can we workshop a better name than `AppState` (it's extremely generic/undescriptive). I'd lean towards a documented `suspended: bool` field myself.